packages/docs/docs/components/path.mdx

```diff
@@ -67,10 +96,10 @@ export default makeScene2D(function* (view) {
 });
 ```
 
-### Animating objects along a spline
+### Animating objects along a path
 
-Like Spline and Bezier, you can also make other components follow this component
-path. You can use the `getPointAtPercentage` to get point along this path.
+Like all nodes extending the [`Curve`][curve] class, the `Path` node has a
```

Comment:
``[`Curve`][curve]`` is a [reference-style link](https://www.markdownguide.org/basic-syntax/#reference-style-links), it assumes that the actual link to `curve` is defined somewhere at the bottom of the file:
```md
[curve]: /api/2d/components/Curve
```